clang-tools-extra/clang-tidy/modernize/MinMaxUseInitializerListCheck.cpp

```diff
@@ -80,27 +80,26 @@ static SmallVector<FixItHint>
 generateReplacements(const MatchFinder::MatchResult &Match,
                      const CallExpr *TopCall, const FindArgsResult &Result,
                      const bool IgnoreNonTrivialTypes,
-                     const unsigned long IgnoreTrivialTypesOfSizeAbove) {
+                     const long IgnoreTrivialTypesOfSizeAbove) {
```

Comment:
avoid using things like long/int. We got std::uint64_t, std::uint32_t for these things.